src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -597,4 +599,21 @@ public function removeParameter($parameterName)
             }
         }
     }
+
+    /**
+     * Fetch an array of all return statements found within this function.
```

Comment:
What about return statements in closures and anonymous classes defined in this method?